artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/jdbc/JdbcLeaseLock.java

```diff
@@ -147,14 +148,13 @@ private long dbCurrentTimeMillis(Connection connection) throws SQLException {
          try (ResultSet resultSet = currentDateTime.executeQuery()) {
             resultSet.next();
             final long endTime = stripMilliseconds(System.currentTimeMillis());
-            final Timestamp currentTimestamp = resultSet.getTimestamp(1);
-            final long currentTime = currentTimestamp.getTime();
-            final long currentTimeMillis = stripMilliseconds(currentTime);
-            if (currentTimeMillis < startTime) {
-               LOGGER.warnf("[%s] %s query currentTimestamp = %s on database should happen AFTER %s on broker", lockName, holderId, currentTimestamp, new Timestamp(startTime));
+            final long currentTime = resultSet.getTimestamp(1, Calendar.getInstance(TimeZone.getTimeZone("UTC"))).getTime();
+            final long currentTimeNoMillis = stripMilliseconds(currentTime);
+            if (currentTimeNoMillis < startTime) {
```

Comment:
Shouldn't this method be called something like verifyAndGetDatabaseCurrentTimestamp to better reflect the internal logic?

Reply:
Given that verification here won't imply anything but some output message probably `verify` can be omitted (unless I'll decide to perform a proper validation with thrown exception)